packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/testable/ServiceTestEditorState.ts

```diff
@@ -333,11 +334,17 @@ export class ServiceTestSetupState {
         label: getSerializationFormatLabel(test.serializationFormat),
       };
     }
-    return undefined;
+    return {
+      value: SERIALIZATION_FORMAT.DEFAULT,
+      label: SERIALIZATION_FORMAT.DEFAULT,
+    };
   }
 
   changeSerializationFormat(val: string | undefined): void {
-    service_setSerializationFormat(this.testState.test, val);
+    service_setSerializationFormat(
+      this.testState.test,
+      val === SERIALIZATION_FORMAT.DEFAULT ? undefined : val,
```

Comment:
May i know why we set it to undefined when it's default?

Reply:
The intention was engine takes `DEFAULT` format even if we don't send anything. This check is unnecessary. I will remove this